frontend/app_flowy/lib/workspace/presentation/settings/widgets/settings_appearance_view.dart

```diff
@@ -28,6 +30,76 @@ class SettingsAppearanceView extends StatelessWidget {
   }
 }
 
+class ThemeTypeSetting extends StatelessWidget {
```

Comment:
Same as above, ThemeType seems out of place

Reply:
Okay